allauth/account/adapter.py

```diff
@@ -483,15 +484,11 @@ def set_password(self, user, password):
 
     def get_user_search_fields(self):
         user = get_user_model()()
-        return filter(
-            lambda a: a and hasattr(user, a),
-            [
-                app_settings.USER_MODEL_USERNAME_FIELD,
-                "first_name",
-                "last_name",
-                "email",
-            ],
-        )
+        return [
+            f.name
+            for f in user._meta.fields
+            if type(f) in [CharField, EmailField] and f.name != "password"
```

Comment:
The changes made here don't seem to be related to the PR title:

>  feat(account): Add unique constraint for a user's primary email #2996

Reply:
Pretty nonsensical PR - close this.  Looks like they crossed feature branches or something...